test/Tests/src/Data/Text_Spec.enso

```diff
@@ -274,7 +274,29 @@ spec =
             utf_8_whitespace.split "\s+" use_regex=True . should_equal utf_8_whitespace_split
             'abc  def\tghi'.split '\\s+' use_regex=True . should_equal ["abc", "def", "ghi"]
 
-        Test.specify "exmples should be correct" <|
+        Test.specify 'should be able to split with a vector of strings' <|
+            'azbzczdzezfzg'.split ['b', 'zez'] . should_equal ['az', 'zczd', 'fzg']
+            'blah x 123'.split [' ', ' x ' , 'x'] . should_equal ['blah', '', '', '123']
+            'a1b2c3d4e5f6g7h8'.split ['c', '5'] . should_equal ['a1b2', '3d4e', 'f6g7h8']
```

Comment:
I think ideally split should work consistently with Unicode.

Could you add these tests?
```
'aśbs\u0301c'.split 'ś' . should_equal ['a', 'b', 'c']
'aśbs\u0301c'.split ['ś'] . should_equal ['a', 'b', 'c']
'aśbs\u0301c'.split 's\u0301' . should_equal ['a', 'b', 'c']
'aśbs\u0301c'.split ['s\u0301'] . should_equal ['a', 'b', 'c']
'aśbs\u0301cdef'.split ['ś', 'de'] . should_equal ['a', 'b', 'c', 'f']
```

Reply:
Yes - @radeusgd is correct. 
@GregoryTravis and I spoke yesterday about using the Regex based split as quicker to implement.
I think we will need to go to a slower approach as we discussed yesterday.

If the delimiter is a length 1 vector we should fall to the standard approach.